doc/man3/OSSL_print_attribute_value.pod

```diff
@@ -0,0 +1,55 @@
+=pod
+
+=head1 NAME
+
+OSSL_print_attribute_value
+- Print an X.500 directory attribute value
```

Comment:
X.509, no ?

Reply:
I said X.500 because X.509 is a part of a larger X.500-series of specifications in which attribute types and values are introduced. Technically, the attributes used in these extensions are actually introduced in X.501. X.509 is just for cryptography / PKI / security aspects of the series.

Reply:
That said, I am sure others would have similar confusion, so I'd be happy to change this if you still insist.

Reply:
I will interpret your thumbs-up as assent and mark this issue resolved. It might not matter anymore, since I am going to make `OSSL_print_attribute_value` a private function now and remove this documentation.